### DISCUSSION TITLE
New Figma UI kit to match the 3.x updates

### DISCUSSION THREAD

**Discussion by QuietSpecialist** [primefaces/community#1609] [Figma UI Kit]
Hi,

Apologies if this has been asked already, but I noticed some significant updates to the PrimeVue components with a recent 3.x release.

For example, the Figma UI kit SelectButton is a closer match to the Vue 2.x component than the 3.51 equivalent.

I also noticed there's a v4 beta now too!

I appreciate these Figma kits take time to build, and the version I have is great btw, but do you have an idea of when we can expect a new version to match the 3 (or 4) component libraries, please?

**Comment by onursenture** [comment]
Yes, the changes you mentioned relate to our new theme, Aura. The Aura theme will be included in future PrimeOne releases. We are continuing to work on this. We will share the Aura theme with PrimeOne users as soon as possible. Thank you.

**Comment by slavco86** [comment]
Will this also include supporting `variant` prop on the Checkbox component, which is currently missing from checkbox definition in 2.2.0?
![Screenshot 2024-04-16 at 12 49 30](https://github.com/primefaces/community/assets/25765449/fa237118-c75f-4130-ab02-fd497f50c5e0)
![Screenshot 2024-04-16 at 12 49 05](https://github.com/primefaces/community/assets/25765449/daa53fd1-50e4-4379-aa83-2e055b4af081)

**Reply by onursenture** [reply]
Hello,

We have noted in the backlog that we need to make the necessary additions to components that need to include the 'filled' property. We will be adding this in upcoming versions.

Thank you for your feedback,

**Reply by slavco86** [reply]
Can I ask roughly when is the next update planned for?

Our company is considering adopting Prime Kit and PrimeVue tools on a wide basis because of the tight and accurate sync between the two, so would be really helpful to understand at least rough estimates on when the two can be back in sync, please. 🙏

**Reply by onursenture** [reply]
Hello, we cannot provide an estimate at the moment because we are still working on the scope of the next UI Kit.

There are two possibilities; the first one is that the new UI Kit version may be compatible with the new theming API coming with PrimeVue v4, which makes estimating difficult due to many factors influencing the development time, including team participation.

The second possibility is to not wait for all the details of the new theming API for the Figma UI Kit and make an intermediate release like 2.3.0. This release could include our new theme from v3, Aura, and solutions to major feedback received since 2.2.0. In this case, we can provide an estimate by the end of May at the latest.

We will keep you informed of developments. You can also follow the changes on the https://primevue.org/roadmap page and plan accordingly.

**Reply by slavco86** [reply]
Awesome, thanks for such a clear response. Very much appreciate it. This will influence our decision making into possibly obtaining a pro-support licence with your company 🙏

**Reply by slavco86** [reply]
And I personally, would advocate for the Second Option - IMO it makes much more sense to support a much wider user base with improvements quicker. Those,  who use and will keep using V3 for a significant amount of time (just because V4 will get released, doesn't mean every team will immediately jump and migrate to it right that moment)

**Reply by onursenture** [reply]
I absolutely agree; indeed, the likelihood of the second possibility occurring is much higher than the first one. Thank you for your feedback. 🙏

**Comment by jennybird87** [comment]
Hello! I tried to find the answer for this, but was not successful so far: what is the best way for us UX designers to get informed about an update in the Prime NG UI Kit for Figma?

**Reply by onursenture** [reply]
Hello. We announce updates regarding the Figma UI Kit on our [Twitter](https://twitter.com/prime_ng) and [Discord](https://discord.gg/gzKFYnpmCY). Also, you can stay informed about important announcements by subscribing to our [newsletter](https://www.primefaces.org/newsletter).
